shared/components/GoogleMap/Sidebar/Sidebar.spec.js

```diff
@@ -0,0 +1,64 @@
+import React from 'react'
+import { shallow } from 'enzyme'
+
+import { Sidebar } from './Sidebar'
+
+describe('Given a sidebar component', () => {
+  describe('when no spots are passed', () => {
+    it('should render none Spotcards', () => {
+      const props = {
+        sideBarState: {}
+      }
+      const component = shallow(<Sidebar {...props} />)
+
+      expect(component.find('SpotCard').length).toBe(0)
+      component.unmount()
+    })
+  })
+  describe('when spotsToRender have some lenght', () => {
+    it('should render as many sportcards as spot passed', () => {
+      const props = {
+        sideBarState: {
+          spotsToRender: [{ id: 'first' }, { id: 'second' }]
+        }
+      }
+      const component = shallow(<Sidebar {...props} />)
+
+      expect(component.find('SpotCard').length).toBe(props.sideBarState.spotsToRender.length)
+      component.unmount()
```

Comment:
You can stop doing
```
component.unmount()
```

btw. You don't need to unmount it after every test since you're not testing unmonting functionality.